src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+Data Log
+
+By: Norma Chang
+"""
+import json
+import time
+
+def log(data, data_type):
```

Comment:
We need a more descriptive name than data_type. The way you have it set up `(data, data_type)` I think that the data_type is the type of the first data parameter, which is a dictionary.

Reply:
changed name